tensorflow_addons/utils/keras_utils_test.py

```diff
@@ -30,11 +30,13 @@ def test_normalize_tuple(self):
                          keras_utils.normalize_tuple((2, 1, 2),
                                                      n=3,
                                                      name='strides'))
+        # with self.assertRaises(TypeError):
```

Comment:
Why to comment this two lines?

Reply:
I think it may be a duplicate. How do you think?

Reply:
Please remove it if it's duplicated.